website/docs/transferable-record/preparing-transferable-record.md

```diff
@@ -62,6 +62,8 @@ Remember to replace the `tokenRegistry` value with your token registry smart con
 Notice the difference between a transferable record and a verifiable document is the use of `tokenRegistry` instead of `documentStore` in declaring the smart contract address.
 
 > For transferable record, you may not batch process it with other documents. Your directory `raw-documents` may only contain one file.
+> 
+> Otherwise, this will result in a scenario where multiple documents being tied to a single Title Escrow Contract, preventing changes to the Beneficiary and Holder addresses for the individual documents in that batch. 
```

Comment:
close but slightly inaccurate - the title escrow contract is just a construct to enable constraints on transfers

the actual mechanism of transferable record is done by the token registry (which conforms to ethereum ERC721 specification), so the violation here is that the MLETR requires a 1-1 mapping between a token id and the asset ownership record. if batching happens then it will be a 1-many mapping from token id to asset records

Reply:
Thanks @rjchow!🙇‍♂️  I have updated the reason for this part. Let me know if it is more accurate now.😀